bin/emqx

```diff
@@ -202,6 +281,26 @@ generate_config() {
         # the vm, we need to pass it in twice.
         CONFIG_ARGS=" -config $RUNNER_ETC_DIR/app.config -args_file $RUNNER_ETC_DIR/vm.args -vm_args $RUNNER_ETC_DIR/vm.args "
     else
+        for VAR in $(compgen -e); do
+             if echo "$VAR" | grep -q '^EMQX_'; then
+                 VAR_NAME=$(echo "$VAR" | sed -e 's/^EMQX_//' -e 's/__/./g' | tr '[:upper:]' '[:lower:]' | tr -d '[:cntrl:]')
+                 VAR_VALUE=$(echo "${!VAR}" | tr -d '[:cntrl:]')
+                 # ignore non-conf values from docker-entrypoint.sh
+                 if [[ $VAR_NAME != "port"* ]] &&
+                    [[ $VAR_NAME != "service"* ]] &&
+                    [[ $VAR_NAME != "node_name" ]] &&
+                    [[ $VAR_NAME != "node" ]] &&
+                    [[ $VAR_NAME != "host" ]] &&
+                    [[ $VAR_NAME != "wait_time" ]] &&
+                    [[ $VAR_NAME != "name" ]] &&
+                    [[ $VAR_NAME != *"max_clients" ]] &&
+                    [[ $VAR_NAME != "admin_password" ]] &&
+                    [[ $VAR_NAME != "_log_console" ]]; then
```

Comment:
Is it true, if in the future we have added some more variables in docker entrypoint,
it may leak into `emqx.conf.override` and cause a load error ?

This could be quite error prone and hard to troubleshoot.

we can perhaps write a escript to generate the override file,
in which we make use of the schema file to pick var names.